sites/shared/hooks/use-bookmark-docs.mjs

```diff
@@ -0,0 +1,30 @@
+// Hooks
+import { useCallback } from 'react'
+import { useDynamicMdx } from 'shared/hooks/use-dynamic-mdx.mjs'
+// Components
+import { MdxWrapper } from 'shared/components/wrappers/mdx.mjs'
+
+export const useBookmarkDocs = (locale) => {
+  const loader = (option) =>
+    useCallback(
+      () =>
+        import(
+          /* webpackInclude: /docs\/site\/bookmarks\/[a-z]+\/[a-z][a-z]\.md$/ */ `markdown/docs/site/bookmarks/${option}/${locale}.md`
```

Comment:
I have changed it to `orgmarkdown` as I guess it's better to be explicit.
I've kept the magic comment though because it doesn't hurt and I think it's better to have this here so people looking at the code appreciate it exists.